Code/GraphMol/FindRings.cpp

```diff
@@ -1098,112 +1096,112 @@ int symmetrizeSSSR(ROMol &mol) {
   return symmetrizeSSSR(mol, tmp);
 };
 
+static bool shareABond(INT_VECT& ring0, INT_VECT& ring1)
```

Comment:
Can this be replaced by `std::find_first_of` ?

Reply:
Thanks! I was looking for that.